doc/oper.xml

```diff
@@ -1387,3 +1387,45 @@ gap> DigraphShortestDistance(gr, [1, 3], [3, 5]);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="PartialOrderDigraphMeetOfVertices">
+<ManSection>
+  <Oper Name="PartialOrderDigraphMeetOfVertices" Label="for a digraph and two vertices"
+        Arg="digraph, u, v"/>
+  <Oper Name="PartialOrderDigraphJoinOfVertices" Label="for a digraph and two vertices"
+        Arg="digraph, u, v"/>
+  <Returns>A positive integer or <K>fail</K></Returns>
+    <Description>
+    If the first argument is a partial order digraph <Ref Prop="IsPartialOrderDigraph"/>
+    then these operations return the meet, or the join, of the two input vertices. If
+    the meet (or join) is does not exist or is not unique then <K>fail</K> is returned.
```

Comment:
Get rid of "or is not unique" since this is included in the definition of a meet/join.